### PR TITLE
UINV-331 Display past payments for a POL on an invoice line

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UINV-331
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceLineDetails/InvoiceLineDetails.js [hunk 1]
```diff
@@ -160,6 +162,14 @@ const InvoiceLineDetails = ({
                 currency={currency}
               />
             </Accordion>
+            {Boolean(otherRelatedInvoiceLines.length) && (
+              <Accordion
+                id={SECTIONS_INVOICE_LINE.otherRelatedInvoiceLines}
+                label={<FormattedMessage id="ui-invoice.otherRelatedInvoiceLines" />}
+              >
+                <OtherRelatedInvoiceLines invoiceLines={otherRelatedInvoiceLines} />
```
Comment (NikitaSedyx) [inline src/invoices/InvoiceLineDetails/InvoiceLineDetails.js:170]: can otherRelatedInvoiceLines be fetched in OtherRelatedInvoiceLines?

Comment (usavkov-epam) [reply src/invoices/InvoiceLineDetails/InvoiceLineDetails.js]: Yes, made changes

File: src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      to={{
+        pathname: `/invoice/view/${invoiceLine.invoice?.id}/line/${invoiceLine.id}/view`,
+        search,
+      }}
+    >
+      {`${invoiceLine.invoice?.folioInvoiceNo}-${invoiceLine.invoiceLineNumber}`}
+    </Link>
+  ),
+  [COLUMN_INVOICE_DATE]: invoiceLine => <FormattedDate value={invoiceLine.invoice?.invoiceDate} />,
+  vendorName: invoiceLine => invoiceLine.vendor?.name,
```
Comment (AndreiShumski2) [inline src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js:56]: vendor name could be undefined

File: src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  vendorInvoiceNo: invoiceLine => invoiceLine.invoice?.vendorInvoiceNo || <NoValue />,
+  status: invoiceLine => <FormattedMessage id={`ui-invoice.invoice.status.${invoiceLine.invoiceLineStatus?.toLowerCase()}`} />,
```
Comment (AndreiShumski2) [inline src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js:58]: The same if smth wrong with status.. mb provide a default message

**Review by AndreiShumski2 — APPROVED** [review]

File: src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js (hunk 2 above)
Comment (usavkov-epam) [reply src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js]: updated

File: src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js (hunk 3 above)
Comment (usavkov-epam) [reply src/invoices/InvoiceLineDetails/OtherRelatedInvoiceLines/OtherRelatedInvoiceLines.js]: updated

**Review by NikitaSedyx — APPROVED** [review]

**Pull request merged**